addons/account/static/src/components/char_with_placeholder_field/char_with_placeholder_field.js

```diff
@@ -0,0 +1,46 @@
+/** @odoo-module **/
```

Comment:
This will be merged by this pr: https://github.com/odoo/odoo/pull/178588

Reply:
I didn't review it then